src/knexfile.js

```diff
@@ -73,7 +73,7 @@ module.exports = {
   },
   // Test with Oracle on Linux (e.g. on CircleCI)
   // Set environment variables
-  // NOTARY_DB_ENV=oracledci LD_LIBRARY_PATH='/opt/oracle/instantclient'
+  // LEDGER_DB_KNEX_ENV=oracledci LD_LIBRARY_PATH=/opt/oracle/instantclient
```

Comment:
We should probably require that `sqlplus` is in `$PATH` and create a standardized setup process, but I think that can be a separate task.